README.md

```diff
@@ -1,70 +1,147 @@
+- Have a basic knowledge of `React` and `OOP` concept
+- Have a general understanding of what testing is
+- Have `visual-studio code` or any other code editor installed on your computer.
 
-## Learn More
+### Setup
+- In order to get a copy of this project you need to download it from https://github.com/Profsain/hello-react-frontend
+- Extract the zipped file and open it in your code editor
+### Install
+- Run the command bellow in your terminal to get all required files 
+``` Ruby
+npm install
+```
 
-You can learn more in the [Create React App documentation](https://facebook.github.io/create-react-app/docs/getting-started).
+### Run Server
+- Run the command bellow in your terminal
+  rails server
+```React
+npm start
+```
+### Run tests
+you can run one of the following command in your terminal
+1. Run testing
+```React
+npm test
+```
+2. Run linters
+```React
+npm run lint
+```
+## Authors
 
-To learn React, check out the [React documentation](https://reactjs.org/).
+👤 **Husseini Mudi Profsain**
 
-### Code Splitting
+- GitHub: [@githubhandle](https://github.com/Profsain)
+- Twitter: [@twitterhandle](https://twitter.com/profsain)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/profsain)
 
-This section has moved here: [https://facebook.github.io/create-react-app/docs/code-splitting](https://facebook.github.io/create-react-app/docs/code-splitting)
+## 🤝 Contributing
 
-### Analyzing the Bundle Size
+Contributions, issues, and feature requests are welcome!
 
-This section has moved here: [https://facebook.github.io/create-react-app/docs/analyzing-the-bundle-size](https://facebook.github.io/create-react-app/docs/analyzing-the-bundle-size)
+Feel free to check the [issues page](https://github.com/Profsain/hello-react-frontend/issues/).
 
-### Making a Progressive Web App
+## Show your support
 
-This section has moved here: [https://facebook.github.io/create-react-app/docs/making-a-progressive-web-app](https://facebook.github.io/create-react-app/docs/making-a-progressive-web-app)
+Give a ⭐️ if you like this project!
 
-### Advanced Configuration
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-This section has moved here: [https://facebook.github.io/create-react-app/docs/advanced-configuration](https://facebook.github.io/create-react-app/docs/advanced-configuration)
+<!-- ACKNOWLEDGEMENTS -->
 
-### Deployment
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
 
-This section has moved here: [https://facebook.github.io/create-react-app/docs/deployment](https://facebook.github.io/create-react-app/docs/deployment)
+I would like to thank Microverse 
 
-### `npm run build` fails to minify
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-This section has moved here: [https://facebook.github.io/create-react-app/docs/troubleshooting#npm-run-build-fails-to-minify](https://facebook.github.io/create-react-app/docs/troubleshooting#npm-run-build-fails-to-minify)
+## 📝 License
+
+This project is [MIT]() licensed.
```

Comment:
- [ ] Kindly provide the link to the MIT license to make your README look more professional.